data/announcements/scheduled.yaml

```diff
@@ -174,3 +174,23 @@ announcements:
     k8s.gcr.io image registry is gradually being redirected to registry.k8s.io (since Monday March 20th).<br>
     All images available in k8s.gcr.io are available at registry.k8s.io.</br>
     Please read our [announcement](/blog/2023/03/10/image-registry-redirect/) for more details.
+
+- name: Freezing the legacy package repositories - Before
+  startTime: 2023-09-01T00:00:00
+  endTime: 2023-09-13T19:59:59
+  style: "background: #c70202"
+  title: Changes to Linux package downloads for Kubernetes
+  message: |
+    The legacy Google-hosted package repositories (`apt.kubernetes.io` and `yum.kubernetes.io`)<br/>
```

Comment:
I would chop _Google-hosted_

Kubernetes is old enough and has had such rapid adoption that we should assume no one knew who was hosting anything.

The subdomains are necessary and I appreciate italicizing them.